readthedocs/doc_builder/backends/mkdocs.py

```diff
@@ -108,20 +108,26 @@ def append_conf(self, **__):
         docs_path = os.path.join(self.root_path, docs_dir)
 
         # RTD javascript writing
-        rtd_data = self.generate_rtd_data(docs_dir=docs_dir)
+        rtd_data = self.generate_rtd_data(docs_dir=docs_dir, mkdocs_config=user_config)
         with open(os.path.join(docs_path, 'readthedocs-data.js'), 'w') as f:
             f.write(rtd_data)
 
-    def generate_rtd_data(self, docs_dir):
+    def generate_rtd_data(self, docs_dir, mkdocs_config):
         """Generate template properties and render readthedocs-data.js."""
+        # Get the theme name
+        theme_name = 'readthedocs'
+        theme_dir = mkdocs_config.get('theme_dir')
+        if theme_dir:
+            theme_name = theme_dir.rstrip('/').split('/')[-1]
```

Comment:
I could also see this just being `theme` in practice, for example `docs/custom/theme` or something. Should we be slugifying the entire path name here, instead of just taking the last attribute? I guess for what we're doing it's close enough, but it does still feel lossy.